app/posts/teacher-success/2025-06-13-ab-testing-prototype.md

```diff
@@ -0,0 +1,77 @@
+---
+title: A/B testing design concepts on a prototype 
+description: Optimising the prototype to facilitate A/B testing to discern the impact of brand recognition.
+date: 2025-06-13
+tags:
+  - prototyping
+  - trainees
+---
+
+We wanted to A/B test two designs of the proposed service to see what inpact the teacher services branding and logo would have on engagement with content.
+
+## What prompted this work
+
+We recently [tested concepts for a trainee support hub](/teacher-success/testing-concepts/), and wanted to create a hi-fidelity prototype based on feedback and internal discussions that would allow people to browse freely across the support pages.
+
+We have recently created two designs, one that follows the DFE design look and feel (test A) and the second (test B) introducing design elements from the Get into teaching (GIT) service on top of the DFE design. The primary reason for this is to test if these design elements are familiar to trainee teachers and if this service can be seen as a continuation of the service line supporting teachers.
+
+The differences between test A and B include:
+
+* only having the teaching every lesson shapes a life logo on the version B homepage
+
+* only having the Get into teaching colour palette on elements on the version B homepage
+
+* only having the Get into teaching block colour headings on the version B homepage
+
+* only having the Get into teaching block colour headings on the version B homepage
+
+* only haviny a banner with an image and Get in to teaching headers on the version B article category pages
+
+* only having block headers and yellow inset elements on version B article pages
+
+We are interested to see if the influence of the subtle design changes has an impact on the experience when using the service.
+
+### Home page concepts
+
+Homepage test A, based on the DFE design guidance.
+
+![test A homepage using the DFE branding](testahomepage.png)
+
+Homepage test B, based on the DFE design guidance but including elements of Get into teaching.
+
+![test B homepage using the DFE branding](testbhomepage.png)
+
+### Category page concepts
+
+Category pages in test A, based on the DFE design guidance.
+
+![test A homepage using the DFE branding](testacategorypage.png)
+
+Category pages in test B, based on the DFE design guidance but including a banner with an image and block text.
+
+![test B homepage using the DFE branding](testbcategorypage.png)
+
+### Article page concepts
+
+Article pages in test A, based on the DFE design guidance.
+
+![test A homepage using the DFE branding](testacontentpage.png)
+
+Article pages in test B, based on the DFE design guidance but including Get into teaching block headers and inset elements.
+
+![test B homepage using the DFE branding](testbcontentpage.png)
+
+
```

Comment:
The linter is failing because of this extra line. Usually, a workflow fixes these sorts of things.

```suggestion
```